package.json

```diff
@@ -4,8 +4,8 @@
   "description": "The AsyncAPI generator. It can generate documentation, code, anything!",
   "main": "./lib/generator.js",
   "bin": {
-    "asyncapi-generator": "./cli.js",
-    "ag": "./cli.js"
+    "asyncapi-generator": "cli.js",
+    "ag": "cli.js"
```

Comment:
was this change necessary?

Reply:
This change is not necessary ,changing it back to original.